gnuradio-runtime/lib/vmcircbuf_createfilemapping.cc

```diff
@@ -129,12 +138,11 @@ vmcircbuf_createfilemapping::vmcircbuf_createfilemapping(int size) : gr::vmcircb
     }
 
 #ifdef DEBUG
-    fprintf(stderr,
-            "gr::vmcircbuf_mmap_createfilemapping: contiguous? mmap %p %p %p %p\n",
-            (char*)d_first_copy,
-            (char*)d_second_copy,
-            size,
-            (char*)d_first_copy + size);
+    std::stringstream info_msg;
```

Comment:
OK, this change is correct w.r.t. the scope of this PR. But it seems odd: Why not make this a debug-level log, and remove the ifdef? Not a question for you, @marcusmueller, more of a general head-scratcher.